tests/snuba/api/endpoints/test_organization_event_details.py

```diff
@@ -191,9 +191,9 @@ def test_event_links_with_date_range(self):
                 url, format="json", data={"field": ["message", "count()"], "statsPeriod": "7d"}
             )
         assert response.data["eventID"] == "b" * 32
-        assert response.data["nextEventID"] == "c" * 32, "c is newer & matches message + range"
-        assert response.data["previousEventID"] == "2" * 32, "d is older & matches message + range"
-        assert response.data["oldestEventID"] == "2" * 32, "3 is outside range, no match"
+        assert response.data["nextEventID"] == "c" * 32, "c is newer & matches message"
+        assert response.data["previousEventID"] == "2" * 32, "d is older & matches message"
+        assert response.data["oldestEventID"] == "3" * 32, "3 is oldest matching message"
```

Comment:
This change is in line with the way next/prev is calculated. We do not show a time picker on the page driven by this endpoint since it is not meant to be time range aware.